katello/katello-backup

```diff
@@ -35,6 +41,11 @@ optparse = OptionParser.new do |opts|
     @options[:logical_backup] = logical
   end
 
+  opts.on("--features foo,bar", Array, "#{FOREMAN_PROXY.capitalize} features to include in the backup, please specify a list with commas. " \
```

Comment:
the features will actually be on katello servers as well as they have default foreman proxies. I'll update to something more like `--features [FEATURE_LIST]` similar to `--incremental`